packages/twenty-shared/src/utils/aggregateOperations/isCompositeFieldMetadataType.util.ts

```diff
@@ -0,0 +1,22 @@
+import { FieldMetadataType } from 'src/types/FieldMetadataType';
+
+export const isCompositeFieldMetadataType = (
```

Comment:
In a PR right after I will replace all usages of isCompositeFieldMetadataType with this one. I just did not want to overload the PR